CppExport/src/exporter/CppExporter.cpp

```diff
@@ -206,7 +206,13 @@ QList<CodeComposite*> CppExporter::mergeUnits(QList<CodeUnit*>& units, QHash<QSt
 		}
 	}
 
-	return nameToCompositeMap.values();
+	QList<CodeComposite*> result = nameToCompositeMap.values();
+	for (auto potentialMacroData : result)
```

Comment:
Please a put a comment here on top to explain briefly what's going on. (why is the code below necessary)